app/templates/components/post-header.hbs

```diff
@@ -1,2 +1,2 @@
 <div class="post-icon"></div>
-{{post-title post=post}}
+{{post-title post=post saveTitle=saveTitle}}
```

Comment:
There's actually no reason not to call this `save`. We're not just saving one thing. We're only setting one thing and then saving.